Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -72,15 +74,22 @@ void DisplayPropertiesWidget::create_frame()
 
     auto* background_content = new GWidget(background_splitter);
     background_content->set_layout(make<GBoxLayout>(Orientation::Vertical));
-    background_content->layout()->add_spacer();
+    //background_content->layout()->add_spacer();
```

Comment:
Let's not leave commented-out code in the tree if we can avoid it.
If you want to keep it around, you can put it inside `#ifdef DEBUG_LALALA` or something like that.

Reply:
> Let's not leave commented-out code in the tree if we can avoid it.

Ahh my bad, forgot that was there sorry.